data/json/proficiencies/chemistry.json

```diff
@@ -8,5 +28,65 @@
     "default_time_multiplier": 2,
     "default_fail_multiplier": 2.5,
     "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_mutagen",
+    "name": { "str": "Mutagen Production" },
+    "description": "Knowledge on the production of general mutagens.",
+    "can_learn": true,
+    "default_time_multiplier": 1.5,
+    "default_fail_multiplier": 2.5,
+    "time_to_learn": "12 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_serum",
+    "name": { "str": "Serum Production" },
+    "description": "Knowledge on the synthesization of the key to the perfect self.",
+    "can_learn": true,
+    "default_time_multiplier": 2.5,
+    "default_fail_multiplier": 5,
+    "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_fuel",
+    "name": { "str": "Fuel Production" },
+    "description": "Knowledge on how to produce and synthesize fuels, unfortunately this won't help make gasoline.",
+    "can_learn": true,
+    "default_time_multiplier": 2,
+    "default_fail_multiplier": 2.5,
+    "time_to_learn": "6 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_upper",
+    "name": { "str": "Stimulant Production" },
+    "description": "An understanding of how to produce both natural and synthetic stimulants.",
+    "can_learn": true,
+    "default_time_multiplier": 2.5,
+    "default_fail_multiplier": 2.5,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_downer",
+    "name": { "str": "Depressant Production" },
+    "description": "An understanding of how to produce natural and synthetic depressants.",
+    "can_learn": true,
+    "default_time_multiplier": 2.5,
+    "default_fail_multiplier": 2.5,
+    "time_to_learn": "8 h"
+  },
+  {
+    "type": "proficiency",
+    "id": "prof_adhesive",
```

Comment:
As above.

Reply:
Thank you for the advice, I wasn’t really sure in general what to do initially. So this was kind of an experimental to see how people thought. I’ll update this later in the day (after I’ve eaten and other stuff like that) with these critiques in mind.